libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -183,3 +231,34 @@ def tools_condition(
     if hasattr(ai_message, "tool_calls") and len(ai_message.tool_calls) > 0:
         return "tools"
     return "__end__"
+
+
+class InjectedState(InjectedToolArg):
```

Comment:
Could we add a docstring and include hte path in `prebuilt.md` so it has a ref doc rendered? Include examples